gensim/models/lsimodel.py

```diff
@@ -223,7 +227,8 @@ def empty_like(self):
             An empty copy (without corpus) of the current projection.
 
         """
-        return Projection(self.m, self.k, power_iters=self.power_iters, extra_dims=self.extra_dims)
+        return Projection(self.m, self.k, power_iters=self.power_iters,
+                          extra_dims=self.extra_dims, random_seed=self.random_seed)
```

Comment:
Hanging indent please (here and elsewhere); see https://github.com/RaRe-Technologies/gensim/wiki/Developer-page#code-style

Reply:
Thanks for the feedback. I have tried to have hanging indents and trailing commas wherever possible. Please let me know me if I've missing anything. Thanks!

Reply:
Thanks, but this line (and others) still has vertical indent. Can you convert it to hanging indent?

Reply:
Cool. My bad. Only focused on declarations; totally overlooked the function calls. Latest commit should fix most of these. Lemme know if I have missed more. 

Feel free to ignore this question (or just point me in right direction): Anyway to have my IDE flag this code style for me. (I'm using PyCharm)?

Reply:
Thanks for the fixes. I don't about about IDE checks, sorry.

Reply:
No problem. Thanks!